modules/ztp-sno-du-enabling-workload-partitioning.adoc

```diff
@@ -57,16 +57,16 @@ sh-4.4# pgrep ovn | while read i; do taskset -cp $i; done
 .Example output
 [source,terminal]
 ----
-pid 8481's current affinity list: 0-3
-pid 8726's current affinity list: 0-3
-pid 9088's current affinity list: 0-3
-pid 9945's current affinity list: 0-3
-pid 10387's current affinity list: 0-3
-pid 12123's current affinity list: 0-3
-pid 13313's current affinity list: 0-3
+pid 8481's current affinity list: 2-51,54-102
+pid 8726's current affinity list: 2-51,54-102
+pid 9088's current affinity list: 2-51,54-102
+pid 9945's current affinity list: 2-51,54-102
+pid 10387's current affinity list: 2-51,54-102
+pid 12123's current affinity list: 2-51,54-102
+pid 13313's current affinity list: 2-51,54-102
 ----
 
-. Check that the system applications CPU pinning is correct:
+. Check that the OpenShift infrastructure applications CPU pinning is correct:
```

Comment:
Ah think there's been slight mix up! This (L69) should be reverted to since this is still "system". 

The change is actually for `2`...where it says "Check that the user applications CPU pinning is correct:" should change to "Check that the OpenShift infrastructure applications CPU pinning is correct:" (in this case "ovn" is the OCP app)

Reply:
Ahh sorry about that, should be correct now!